Tasks/MavenAuthenticateV0/mavenutils.ts

```diff
@@ -0,0 +1,200 @@
+import fs = require('fs');
+import Q = require('q');
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
+import stripbom = require('strip-bom');
+import { getSystemAccessToken } from 'artifacts-common/webapi'
+
+import * as xml2js from 'xml2js';
+import * as os from 'os';
+import * as fse from 'fs-extra';
+
+import { getPackagingServiceConnections, ServiceConnection, ServiceConnectionAuthType, UsernamePasswordServiceConnection, TokenServiceConnection, PrivateKeyServiceConnection } from "artifacts-common/serviceConnectionUtils";
+
+const accessTokenString = getSystemAccessToken();
```

Comment:
since this should only be used by getInternalFeedsServerElements, may be consider moving it inside that function?

Reply:
Fixed